### ISSUE TITLE
Remove PWA Category

### ISSUE THREAD

**Issue by adrianaixba** [GoogleChrome/lighthouse#15535]
Issue tracking the removal of the PWA category

- [x] deprecation warning banner
- [x] remove the category from the json result
- [x] remove/rearrange PWA audits
- [x] ensure support for report PWA category rendering post removal
- [x] remove references to documentation in codebase
- [x] Get rid of ServiceWorker gatherer
- [x] Get rid of Manifest/InstallabilityErrors gatherers and ManifestValues computed artifact
- [x] unlist [Lighthouse PWA documentation](https://developer.chrome.com/docs/lighthouse/pwa/) in the report
- [x] clean up PWA Github issues

a nice have, but not a blocker for releasing this - would need some more investigation on this one.
- maybe include a `no_pointless_service_worker` audit

**Comment by adamraine** [comment]
Might be worth going through all of the issues marked "pwa" and either close or merge into this issue: https://github.com/GoogleChrome/lighthouse/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+PWA+label%3Apwa

> remove [Lighthouse PWA documentation](https://developer.chrome.com/docs/lighthouse/pwa/)
> remove references to documentation

We can remove references to documentation in v12.0, but old versions of Lighthouse will always link to the PWA documentation we have. For this reason, I don't think we should remove documentation completely. We should keep the https://developer.chrome.com/docs/lighthouse/pwa/ articles alive if you have the exact URL for an article, but you shouldn't be able to find Lighthouse PWA stuff just clicking through https://developer.chrome.com/.

**Comment by brendankenny** [comment]
We definitely want those URLs to last forever. We have a banner we've used in the past to indicate up front that the audit is no longer maintained and when that happened:

<img width="810" alt="The docs for the first-cpu-idle audit, showing a banner at the top cautioning that the metric was deprecated in Lighthouse 6.0" src="https://github.com/GoogleChrome/lighthouse/assets/316891/337fead4-22a7-45ed-af7b-7c4522f5c168">

https://developer.chrome.com/docs/lighthouse/performance/first-cpu-idle/

Not sure how others feel, but I also prefer to have _some_ navigable path to docs that exist, though it shouldn't be prominent. I made a [metrics graveyard at the bottom of the performance section](https://developer.chrome.com/docs/lighthouse/performance/#retired-metrics) with links to retired metrics, for instance. A whole category is slightly different, but we could e.g. make a separate subpage for retired categories or whatever.
